src/components/QDragAndDrop/QDragAndDrop.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="lists">
+    <div
+        class="columns"
+        v-for="(column) in columns"
+    >
+      <p class="column_title">{{column.title}}</p>
+      <ul>
+        <li
+            class="list-item"
+            draggable="true"
+            v-for="(item) in column.items"
+            :style="{ backgroundColor: color }"
+        >{{item.title}}</li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    columnsOfItems: {
+      type: Array,
+      required: true,
+      default: [
+        { id: 1, title: 'TODO', items: [
+            {id: 1, title: 'Titre 1'},
+            {id: 2, title: 'Titre 2'},
+          ]
+        },
+        { id: 2, title: 'IN PROGRESS', items: [
+            {id: 3, title: 'Titre 3'},
+          ]
+        },
+        { id: 3, title: 'DONE', items: [
+            {id: 4, title: 'Titre 4'},
+          ]
+        },
+      ],
```

Comment:
```suggestion
      default: () => ([
        { id: 1, title: 'TODO', items: [
            {id: 1, title: 'Titre 1'},
            {id: 2, title: 'Titre 2'},
          ]
        },
        { id: 2, title: 'IN PROGRESS', items: [
            {id: 3, title: 'Titre 3'},
          ]
        },
        { id: 3, title: 'DONE', items: [
            {id: 4, title: 'Titre 4'},
          ]
        },
      ]),
```

Il est recommandé d'utiliser des arrows functions pour définir une valeur par défaut